### ISSUE TITLE
Problem with rounding numbers

### ISSUE THREAD

**Issue by MortezaRahmani** [BonneVoyager/node-mt4-zmq-bridge#15]
Hi.
Is there anyway to prevent rounding the numbers of stop-loss and take-profits?
e.g., using the below code I am able to set SL and TP for my EURUSD sell order, but their numbers get rounded when placed in MT4:

```
let SL = 1.08709;
let TP = 1.084131;

zmqBridge.request(
                    mt4zmqBridge.REQUEST_TRADE_MODIFY, 
                    62868155, // Ticket Number
                    1.086350, // Base Price
                    SL, // Stop Loss
                    TP // Take Profit
);
```

In MT4, I see the below numbers set as my SL and TP:
SL = **1.09000**
TP = **1.08000**

Thank you.

**Comment by MortezaRahmani** [comment]
I could fix it by commenting line 441 in MetaTrader4Bridge.mq4
```
// price = NormalizeDouble(price, Digits);
```

**Comment by BonneVoyager** [comment]
Hi, thank you for creating this issue and commenting about the fix. I'm not sure why `Digits` didn't work in this case, but I'll look at this when I get a chance. Could you give me more details about your request, which currency pair, and which MT4 server this is? Thanks!

**Comment by BonneVoyager** [comment]
Hi @MortezaRahmani, thank you for creating this issue and commenting about the fix. I'm not sure why `Digits` didn't work in this case, but I'll look at this when I get a chance. Could you give me more details about your request, which currency pair, and which MT4 server this is? Thanks!

**Comment by MortezaRahmani** [comment]
> Hi @MortezaRahmani, thank you for creating this issue and commenting about the fix. I'm not sure why `Digits` didn't work in this case, but I'll look at this when I get a chance. Could you give me more details about your request, which currency pair, and which MT4 server this is? Thanks!

Hi @BonneVoyager, Thanks for your reply.
The problem was on EURUSD pair inside Amarkets-demo MT4 broker server (ECN account).
I was able to open a new sell order using REQUEST_TRADE_OPEN method, and to set stop-loss/take-profit prices, I called the REQUEST_TRADE_MODIFY method with given values and I noticed the expert is rounding the numbers of stop-loss and take-profit.
My stop-loss was set to 1.08709 and the expert rounded it to 1.09000 which is almost 30 pips above my actual stop.
